Sources/SWBTaskConstruction/ProductPlanning/BuildPlan.swift

```diff
@@ -200,23 +201,19 @@ package final class BuildPlan: StaleFileRemovalContext {
 
         // Compute all of the deferred tasks (in parallel).
         delegate.updateProgress(statusMessage: messageShortening == .full ? "Planning deferred tasks" : "Constructing deferred tasks", showInLog: false)
-        await TaskGroup.concurrentPerform(iterations: productPlanResultContexts.count, maximumParallelism: 10) { i in
+        await TaskGroup.concurrentPerform(iterations: productPlanResultContexts.count, maximumParallelism: mediumParallelism) { i in
```

Comment:
The maximumParallelism is intended to put an upper bound on the number of active tasks rather than fan out as widely as possible, since we fan out at multiple levels of the build planning process. What kind of speedup do you see if this change and the one below are applied in isolation?